BLSQ/iaso
# IA-3102: Permissions : Org unit “view only” permission

Explain what problem this PR is resolving
- Permissions : Org unit “view only” permission
Related JIRA tickets : [IA-3102](https://bluesquare.atlassian.net/browse/IA-3102)

## Self proofreading checklist


## Changes
- Add new Organisation units - Read permission
- Check on create_org_unit and edit org unit the Organisation units management permission
- Hide some button and action if the user has not Organisation units management permission on frontend

## How to test
- Create or update a user and remove the Organisation units management and add Organisation units - Read permission to it
- Check all the actions for create and update org units are possible

## Print screen / video
[Screencast from 2024-08-19 14-23-35.webm](https://github.com/user-attachments/assets/781f2a0b-29a3-4287-a6f6-1f022fe8121d)



## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3102]: https://bluesquare.atlassian.net/browse/IA-3102?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
